package.json

```diff
@@ -5,7 +5,7 @@
     "build": "nuxi build",
     "start": "node .output/server/index.mjs",
     "preview": "nuxi preview",
-    "test": "jest"
+    "test:unit": "vue-cli-service test:unit"
```

Comment:
I think we have to install @vue/cli-service package like we did on the DSv2
https://www.npmjs.com/package/@vue/cli-service

Reply:
oh must have forgotten to save - I added it to the package!